src/musicPlayer.java

```diff
@@ -1,23 +1,23 @@
 import components.queue.Queue;
 import components.queue.Queue1L;
 
-/** */
 public class musicPlayer {
-    /** */
 
     private Queue<String> playlist;
+    private String currentTrack;
+    private boolean isPlaying;
 
-    /** */
     public musicPlayer() {
         this.playlist = new Queue1L<>();
+        this.currentTrack = null;
+        this.isPlaying = false;
     }
 
-    /** */
-    public void queue(String song) {
+    public void addSong(String song) {
```

Comment:
I might expand this idea a bit more. Maybe let the user provide a Map or a Song object. That way, they can customize what information they want to store like artist, title, album, albumArt, etc.